### DISCUSSION TITLE
How to make form element only displayed on the creation page

### DISCUSSION THREAD

**Discussion by xiaomlove** [filamentphp/filament#6327] [Help] [answered]
I want a form form item to be displayed only on the create page, but not on the edit page, how can I do that? I have tried the hidden() method, but I don't know how to determine whether it is currently create or edit. Then I dump all parameters：
```
... ->hidden(fn () => dd(func_get_args()))

```
![image](https://user-images.githubusercontent.com/8530353/234649142-9d2f1c73-090d-44fb-938f-d887b08cfcd2.png)

The second parameter's value is what I want。How can I got it correctly ?

**Comment by danharrin** [comment, marked as answer]
`visibleOn('create')`, its in the docs

**Comment by xiaomlove** [comment]
I saw it, thanks a lot. I couldn't find it in the part of  Form Package, but it was in the Admin package part
